content/docs/strict-mode.md

```diff
@@ -8,6 +8,8 @@ permalink: docs/strict-mode.html
 
 > Note:
 >
+> Strict mode may "_render_" your application twice, but only one will be visible to the DOM. The second render is necessary to carry out its checks.
```

Comment:
The article already mentions this. This now gives the impression this is the only effect StrictMode has. But it has many more effects.

Reply:
Could you point where it in this doc it already specifies that it renders twice? Clearly I am missing it....

Reply:
https://reactjs.org/docs/strict-mode.html#detecting-unexpected-side-effects